Header header
RequestType request_type
---
TeamStatus[] team_list